js/components/sidebar.vue

```diff
@@ -83,6 +83,15 @@ export default {
                 };
             });
         }
+    },
+    methods: {
+        onSearch() {
+            const terms = this.$els.terms.value;
+            if (terms && terms.length > 2) {
+                this.$go({name: 'search', query: { terms: terms }});
+                this.$root.me.$emit('updated');
```

Comment:
Trigger another event from the sidebar which refresh the view, I just want to update results when the search terms are modified in the sidebar.